rxnetty-http/src/test/java/io/reactivex/netty/protocol/http/server/HttpServerRule.java

```diff
@@ -114,4 +183,55 @@ public void setServer(HttpServer<ByteBuf, ByteBuf> server) {
     public HttpClient<ByteBuf, ByteBuf> getClient() {
         return client;
     }
+
+
+    public void getAndDrainClient() {
+        lastResponse = "";
+        TestSubscriber<Void> clientDrain = new TestSubscriber<>();
+        client.createGet("/")
+                .flatMap(new Func1<HttpClientResponse<ByteBuf>, Observable<Void>>() {
+                    @Override
+                    public Observable<Void> call(HttpClientResponse<ByteBuf> clientResponse) {
+                        return clientResponse.discardContent();
+                    }
+                })
+                .subscribe(clientDrain);
+        clientDrain.awaitTerminalEvent();
+        clientDrain.assertNoErrors();
+    }
+
+
+    private static class RawMessageHandler extends ChannelDuplexHandler {
```

Comment:
I've left this in to capture the response since there's no `#channelProvider()` method on the server side.